assets/js/events.js

```diff
@@ -49,8 +49,15 @@ export function registerGlobalEventHandlers() {
 
   window.addEventListener("lb:clipcopy", (event) => {
     if ("clipboard" in navigator) {
+      const value = event.target.value;
       const text = event.target.textContent;
-      navigator.clipboard.writeText(text);
+      const tag = event.target.tagName;
+
+      if (tag === "INPUT") {
+        navigator.clipboard.writeText(value);
+      } else {
+        navigator.clipboard.writeText(text);
+      }
```

Comment:
```suggestion
      if (tag === "INPUT") {
        navigator.clipboard.writeText(event.target.value);
      } else {
        navigator.clipboard.writeText(event.target.tagName);
      }
```

And then get rid of the variables?